apps/server/src/services/RestoreService.ts

```diff
@@ -0,0 +1,106 @@
+import { Playback } from 'ontime-types';
+import { logger } from '../classes/Logger.js';
+import { PlaybackService } from './PlaybackService.js';
+import { eventTimer } from './TimerService.js';
+import { EventLoader } from '../classes/event-loader/EventLoader.js';
+
+//File stuff
+import fs from 'fs';
+import { resolveDbDirectory } from '../setup.js'
+import path from 'path';
+import { Writer } from 'steno';
+
+type OntimeState = {
+    playback: Playback;
+    selectedEventId: string | null;
+    startedAt: number | null;
+    addedTime: number | null;
+    pausedAt: number | null
+}
+
+/**
+ * Service manages saveing of timer state
+ * that can then be resored when reopening
+ */
+export class RestoreService {
+    private lastStore: string = '';
+    private readonly file;
+    private load: Partial<OntimeState> = { playback: Playback.Armed };
+    private readonly ok;
+    private readonly filePath;
```

Comment:
some questions here:
- can this be named more semantically to what they achieve? specifically ok (hasValidPayload?) and load (payload?)
- why is load initialised with armed?